doc/source/cli_topo.md

```diff
@@ -0,0 +1,265 @@
+(cli-topo)=
+
+# Topo workflow
+
+## Summary
+
+The topo workflow is designed to provide users with comprehensive information about their elevation model,
+including terrain attributes (i.e. slope, hillshade, aspect, etc.) and statistical analyses (i.e. mean, max, min, etc.)
+
+
+
+:::{figure} imgs/topo_workflow_pipeline.png
```

Comment:
The text font is a bit small when displayed on the final doc. Would it be possible to increase it?